storage.rules

```diff
@@ -2,7 +2,7 @@ rules_version = '2';
 service firebase.storage {
   match /b/{bucket}/o {
     match /{allPaths=**} {
-      allow read, write: if request.auth!=null;
```

Comment:
Could you explain this decision?

Reply:
The idea was to make sure the code can access the local emulator in tests (those rules are only used available locally I think) but it does not work with it either so....